instrumentation/couchbase/couchbase-3.1/javaagent/couchbase-3.1-javaagent.gradle

```diff
@@ -21,4 +21,9 @@ dependencies {
   testLibrary group: "com.couchbase.client", name: "java-client", version: "3.1.0"
 
   testImplementation group: "org.testcontainers", name: "couchbase", version: versions.testcontainers
-}
\ No newline at end of file
+}
+
+configurations {
+  // exclude lib from test classpath, otherwise the javaagent would be skipped
+  testImplementation.exclude group: 'com.couchbase.client', module: 'tracing-opentelemetry'
+}
```

Comment:
is this a general problem for all javaagent instrumentation that is based on library instrumentation? or only a problem if the javaagent instrumentation depends on 3rd party library instrumentation?

Reply:
No, it's only a problem when you use a 3rd party library instrumentation. Our library instrumentation jars are filtered out in `gradle/instrumentation.gradle`.